scripts/check_regression.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Check if two arguments are provided (paths to past.json and current.json)
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 <path_to_past.json> <path_to_current.json>"
+    exit 1
+fi
+
+# Extract the file paths from command-line arguments
+past_json_path="$1"
+current_json_path="$2"
+
+# Read the contents of past.json and current.json into variables
+past_data=$(cat "$past_json_path")
+current_data=$(cat "$current_json_path")
+
+# Function to compare P90 values for a given statistic
+compare_stat_p90() {
+    local past_value="$1"
+    local current_value="$2"
+    local test_name="$3"
+    local stat_name="$4"
+
+    # Calculate 110% of the past value
+    local threshold=$(awk -v past="$past_value" 'BEGIN { print past * 1.1 }')
+
+    # Compare the current value with the threshold
+    if (( $(awk 'BEGIN {print ("'"$current_value"'" > "'"$threshold"'")}') )); then
+        echo "Test '$test_name' - Threshold: $threshold"
+        echo "ERROR: Test '$test_name' - Current P90 value ($current_value) exceeds 110% of past $stat_name P90 value ($past_value)"
```

Comment:
nit:  echo "ERROR: $stat_name - Current P90 value ($current_value) exceeds the 110% threshold ($threshold) of the past P90 value ($past_value)"

Reply:
Updated